### ISSUE TITLE
Can't scan passports on Android

### ISSUE THREAD

**Issue by eliotstock** [selfxyz/self#192]
Tested with UK and NZ passports on a Pixel 8.

I get the initial vibration, but nothing more.

**Comment by 0xturboblitz** [comment]
Hey, thanks for the feedback.

Have you tried waiting longer after the initial vibration? Also, curious if you got an error message, or saw the progress of the reading displayed on the screen. Feel free to dm me [here](https://t.me/FlorentTavernier) to chat more about that.

**Comment by eliotstock** [comment]
Yep, I stop moving on the initial vibration as instructed. Then, nada. No error message for a bit, then a flash of two or three messages that disappear too quickly to read, then back to the "start scanning" button.

Also, the instructions say to hold the phone over the last page of the passport. That's valid for my UK passport. For my NZ passport, the NFC chip is on the second page from the front.

Would be happy to test new builds as they come out. Will DM you.

**Comment by 0xturboblitz** [comment]
Thanks, will definitely improve that in coming versions.

**Comment by 0xturboblitz** [comment]
Closing this one for now, goes with the general NFC improvement work
